.github/workflows/doc-linkcheck.yml

```diff
@@ -0,0 +1,50 @@
+name: doc-linkcheck
+run-name: Documentation Link Check
+
+on:
+  pull_request:
+    types: [labeled]
+
+permissions:
+  contents: read
+
+env:
+  HATCH_ENV_TYPE_VIRTUAL_PATH: ${{ github.workspace }}/.hatch_envs
+
+jobs:
+  linkcheck-docs:
+    name: Sphinx Documentation Link Check
+    runs-on: ubuntu-latest
+    # Run only when the specific label is added
+    if: github.event.label.name == 'status | ready for merge'
```

Comment:
```suggestion
```

Remove this so it runs as much as possible

Reply:
The check would be either `ci | full` test or `documentation`.